bigframes/dataframe.py

```diff
@@ -1913,6 +1915,11 @@ def _reindex_rows(
     def _reindex_columns(self, columns):
         block = self._block
         new_column_index, indexer = self.columns.reindex(columns)
+
+        if indexer is None:
+            # The new index is the same as the old one. Do nothing.
```

Comment:
I'm curious how you discovered this. Worth separating out into it's own PR (with tests).

Reply:
This was discovered when I wrote an ealier version of the "describe()": 

I didn't check the emptiness of aggregated dataframes, and always concated them in the end. Then there was an edge case where the original dataframe contains only numerical columns, and as a result I was basically reindexing the columns with their original order, which triggered this bug.